docs/modules/storage/pages/high-density-memory.adoc

```diff
@@ -245,120 +249,85 @@ PersistentMemoryConfig pmemConfig = memoryConfig.getPersistentMemoryConfig()
 config.setNativeMemoryConfig(memoryConfig);
 ----
 
-The above snippets demonstrate how to configure the persistent memory as
-High-Density Memory Store in Hazelcast. The example assumes dual-socket machine;
-both sockets are populated with Intel(R) Optane(TM) DC persistent memory DIMMs that
-are configured in interleaved mode. The two sockets' DIMMs are mounted as `/mnt/pmem0`
-and `/mnt/pmem1` and are known as NUMA node0 and node1, respectively.
-
-Here are the descriptions of the  persistent memory configuration elements and attributes:
-
-* `enabled`: Specifies whether the persistent memory usage is enabled or not. Its default value is `false`, 
-i.e., persistent memory usage is disabled.
-* `mode`: Defines the persistent memory operational mode. Two modes are supported:
-** `MOUNTED`: If you choose this mode, the persistent memory is mounted into the file system (aka FS DAX).
-** `SYSTEM_MEMORY`: If you choose this mode, the persistent memory is onlined as system memory (aka KMEM DAX).
-* `directories`: List of the persistent memory mounting directories to store data of all the data structures backed by
-High-Density Memory Store. When you provide the directories:
-** the persistent memory usage is enabled automatically and you do not need to explicitly
+The following elements and attributes are also used in the configuration: 
+
+* `enabled`: Specifies whether use of memory on the persistent memory device is enabled or not. The default value is `false`, that is use of the device is disabled.
+* `mode`: Defines the operational mode of the persistent memory device. Two modes are supported:
+** `MOUNTED`: If you choose this mode, the memory is mounted into the file system (aka FS DAX).
+** `SYSTEM_MEMORY`: If you choose this mode, the memory is onlined as system memory (aka KMEM DAX).
```

Comment:
Actually, the memory has to be already online. The onlining is done at system level, this option just makes HZ using that portion of the memory.

Reply:
I've updated the description but could you have a quick check to make sure that this is still accurate?

Reply:
Maybe "...Hazelcast uses the already onlined PMEM portion of the system memory..." indicates it better that the user has something to do in order to use this mode.

Reply:
Updated as requested.